charts/argo-cd/README.md

```diff
@@ -1226,6 +1226,35 @@ If you want to use an existing Redis (eg. a managed service from a cloud provide
 | notifications.topologySpreadConstraints | list | `[]` (defaults to global.topologySpreadConstraints) | Assign custom [TopologySpreadConstraints] rules to the application controller |
 | notifications.triggers | object | `{}` | The trigger defines the condition when the notification should be sent |
 
+## <a id="reference-other-values"></a>Values that reference other values
+Some value may reference other values.
+When you may use it? For example in case of multiple environments you can use the same argocd values when environment specific
+values are defined in separate environment file that is used by multiple helm charts
+```
+# env1-values.yaml
+global:
+  dnsDomain: .my.domain.com
+```
+
+```
+# env2-values.yaml
+global:
+  dnsDomain: .my.other-domain.com
+```
+
+```
+# argocd's values.yaml
+server:
+  ingress:
+    hosts:
+      - "argocd {{ .Values.global.dnsDomain }}" # be sure protect by quotation marks
+```
+
+```
+helm install my-release argo/argo-cd --context env1 -n argocd -f env/global/configuration/env1-values.yaml
+helm install my-release argo/argo-cd --context env2 -n argocd -f env/global/configuration/env2-values.yaml
+```
```

Comment:
It depends on the values if we can refer other values, so I wonder how we keep updating this section. 🤔 
Maybe not necessary?

Reply:
@yu-croco It's just example that shows how to refer other values. I don't think that the example should show each value that is rendered. However values are evaluated should be highlighted (that is done)
https://github.com/argoproj/argo-helm/pull/2306/files#diff-9b2b8960b2091a28395a78f3f4b76c4310b39062f6602f9a0575a27ed890f904R724